ethers-core/src/types/block.rs

```diff
@@ -84,6 +84,10 @@ pub struct Block<TX> {
     /// Base fee per unit of gas (if past London)
     #[serde(rename = "baseFeePerGas")]
     pub base_fee_per_gas: Option<U256>,
+    /// Withdrawals root hash (EIP-4895)
+    #[serde(rename = "withdrawalsRoot")]
+    #[cfg(not(feature = "celo"))]
+    pub withdrawals_root: Option<H256>,
```

Comment:
this field is not present pre shanghai and should be `skip_serialize_if Option::is_none`